### ISSUE TITLE
psutil.cpu_freq() fails with FileNotFoundError on CentOS 7.3

### ISSUE THREAD

**Issue by SethMichaelLarson** [giampaolo/psutil#1114]
```python
import psutil
psutil.cpu_freq()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/slarson/.python3/bin/lib/python3.5/site-packages/psutil/__init__.py", line 1895, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/home/slarson/.python3/bin/lib/python3.5/site-packages/psutil/_pslinux.py", line 666, in cpu_freq
    curr = int(cat(os.path.join(path, "scaling_cur_freq"))) / 1000
  File "/home/slarson/.python3/bin/lib/python3.5/site-packages/psutil/_pslinux.py", line 298, in cat
    with open_binary(fname) if binary else open_text(fname) as f:
  File "/home/slarson/.python3/bin/lib/python3.5/site-packages/psutil/_pslinux.py", line 190, in open_binary
    return open(fname, "rb", **kwargs)
FileNotFoundError: [Errno 2] No such file or directory: '/sys/devices/system/cpu/cpu0/cpufreq/scaling_cur_freq'
```

Here is the output of `ls -Al /sys/devices/system/cpu/cpu0/cpufreq`:

```bash
$ ls -Al /sys/devices/system/cpu/cpu0/cpufreq
total 0
-r--r--r--. 1 root root 4096 Jul 31 15:32 affected_cpus
-r--------. 1 root root 4096 Jul 31 15:32 cpuinfo_cur_freq
-r--r--r--. 1 root root 4096 Jul 31 15:32 cpuinfo_max_freq
-r--r--r--. 1 root root 4096 Jul 31 15:32 cpuinfo_min_freq
-r--r--r--. 1 root root 4096 Jul 31 15:32 cpuinfo_transition_latency
-r--r--r--. 1 root root 4096 Jul 31 15:32 related_cpus
-r--r--r--. 1 root root 4096 Jul 31 15:32 scaling_available_governors
-r--r--r--. 1 root root 4096 Jul 31 15:32 scaling_driver
-rw-r--r--. 1 root root 4096 Jul 31 15:32 scaling_governor
-rw-r--r--. 1 root root 4096 Jul 31 15:32 scaling_max_freq
-rw-r--r--. 1 root root 4096 Jul 31 15:32 scaling_min_freq
-rw-r--r--. 1 root root 4096 Jul 31 15:32 scaling_setspeed
```

Was wondering if this issue could be fixed somehow? I can give the contents of these files if needed.

**Comment by wiggin15** [comment]
I believe this is a duplicate of #1071 which is fixed in master branch but not yet released.

For further reference, this is documented in a RHEL KB article: https://access.redhat.com/solutions/3116471 (may require RedHat login). Their resolution states: `This fix will be included in Red Hat Enterprise Linux 7.4.`

**Comment by giampaolo** [comment]
Right, this is a duplicate of #1071.
